### PR TITLE
Adds "Toggle Light" to Captain's Spacesuit Helmet

### PR DESCRIPTION
**Pull request by jknpj**
helmlight
Sprites by Blithering

https://webmshare.com/rZQPN

Closes: #13325

:cl:
 * rscadd: Adds "Toggle Light" to Captain's Spacesuit Helmet

### CODE REVIEW

**Comment by Exxion** [comment]
>Toogle

**Comment by jknpj** [comment]
Jesus let me fix that

File: code/modules/clothing/spacesuits/captain.dm [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Only for the most fashionable of military figureheads."
 	body_parts_covered = HEAD|EARS|EYES
 	permeability_coefficient = 0.01
 	pressure_resistance = 200 * ONE_ATMOSPHERE
 	armor = list(melee = 65, bullet = 50, laser = 50,energy = 25, bomb = 50, bio = 100, rad = 50)
+	allowed = list(/obj/item/device/flashlight)
+	light_power = 1.7
+	var/brightness_on = 4
+	var/on = 0
+	var/no_light = 0
```
Comment (9600bauds) [inline code/modules/clothing/spacesuits/captain.dm]: ??

File: code/modules/clothing/spacesuits/captain.dm [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
 	name = "space helmet"
-	icon_state = "capspace"
+	icon_state = "capspace_0"
 	item_state = "capspacehelmet"
 	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Only for the most fashionable of military figureheads."
 	body_parts_covered = HEAD|EARS|EYES
 	permeability_coefficient = 0.01
 	pressure_resistance = 200 * ONE_ATMOSPHERE
 	armor = list(melee = 65, bullet = 50, laser = 50,energy = 25, bomb = 50, bio = 100, rad = 50)
+	allowed = list(/obj/item/device/flashlight)
```
Comment (9600bauds) [inline code/modules/clothing/spacesuits/captain.dm]: ????

**Comment by kol1th** [comment]
Delet this 🔫

**Review by Probe1 — APPROVED** [review]

File: code/modules/clothing/spacesuits/captain.dm [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
+/obj/item/clothing/head/helmet/space/capspace/attack_self(mob/user)
+	on = !on
+	icon_state = "capspace_[on]"
+	user.update_inv_head()
+
+	if(on)
+		set_light(brightness_on)
+	else
+		set_light(0)
+		user.update_inv_head()
```
Comment (ComicIronic) [inline code/modules/clothing/spacesuits/captain.dm]: Why do you call this again if it's called above?

Comment (Probe1) [reply code/modules/clothing/spacesuits/captain.dm]: The whole thing should probably be a ternary but I'm not gonna make him rewrite helmets for it.

**Review by PJB3005 — APPROVED** [review]

**Comment by jknpj** [comment]
Why won't it solve, W H Y

**Comment by Probe1** [comment]
The vox did this.

**Comment by Probe1** [comment]
You did it!

**Comment by Probe1** [comment]
inb4 conflicts again

**Comment by jknpj** [comment]
know i know how to solve it...probably.

**Pull request merged**